src/dkg/key_generation.rs

```diff
@@ -521,7 +521,8 @@ pub struct DKGParticipantList<C: CipherSuite> {
 impl<C: CipherSuite> DistributedKeyGeneration<RoundOne, C> {
```

Comment:
Random question, but if the `to_bytes` function is identical for a `DistributedKeyGeneration<RoundTwo, _>`, can't you implement both at the same time?

Reply:
Yes `to_bytes` / `from_bytes` could be defined for an arbitrary `DkgState`, I didn't bother dealing with it because we may refactor this, following #8. But that's acknowledged, thanks for noticing 👍🏼